get-started/07_multi_container.md

```diff
@@ -110,13 +110,20 @@ address)?
 To figure it out, we're going to make use of the [nicolaka/netshoot](https://github.com/nicolaka/netshoot) container,
 which ships with a _lot_ of tools that are useful for troubleshooting or debugging networking issues.
 
-1. Start a new container using the nicolaka/netshoot image. Make sure to connect it to the same network.
+
+1. Exit MySQL if you haven't already (if you have typed any thing before press `;` and hit enter).
```

Comment:
@YgorPerez Thanks for the suggestion. Instead of adding the exit step here. Could you add it below line 102 so the **Connect to MySQL** section is cleaner? Please use the following text:


Exit the MySQL shell to return to the shell on our machine.

 ```console
    mysql> exit
 ```

Reply:
Sure!